content/create-new-collection.en.md

```diff
@@ -0,0 +1,160 @@
+- Collection name (3 words maximum, examples: `Platform Governance Archive`, `France Élections`, `Dating`, `P2B Compliance Assessment`…)
+- Collection ID (examples: `pga`, `France-elections`, `dating`, `p2b-compliance`…)
+- Terms language (examples: `English`, `French`, `English`, `All EU languages`…)
+- Service jurisdiction (examples: `EU`, `France`, `EEA`, `EU`…)
+- Collection maintainer entity (name, logo, url)
+
+## Create repositories
+
+### Collection definition
+
+Create the collection definition repository by using the [Demo collection](https://github.com/OpenTermsArchive/demo-declarations) as template:
+- Go to the [demo collection repository homepage](https://github.com/OpenTermsArchive/demo-declarations)
+- Click on the "Use this template" dropdown and select "Create a new repository"
+- In the new "Create a new repository" page that opens up,  fill the field `Repository name` by using the collection `ID` followed by `-declarations`. For example: `demo-declarations`
+- When redirected to the newly generated repository, please wait until the automatic setup is done (it should be done in less than a minute). You can check the status of the `first-time-setup` GitHub action to find out if everything went smoothly.
+
+#### Fill the about section
+
+- Click on the little cogwheel icon next to the "About" block.
+
+- Set the description: “Declarations for `<collection_name>`. Maintained by `<maintainer>`.”
+- Set website: https://opentermsarchive.org
+- Add the following tags: `terms-of-service`, `terms-of-service-agreements`, `terms-and-conditions`, `open-terms-archive`.
+- Uncheck “Releases”, “Packages” and “Environments”.
+
+#### Define repository settings
+
+These settings ease the whole contribution process.
+
+- In the `Pull Requests` section: check only the “Allow squash merging” option, and set it to “Default to pull request title and commit details”.
+- Enable “Allow auto-merge”.
+- Enable “Automatically delete head branches”.
+- In the `Branches` section: add a branch protection rule for `main`, check “Require status checks to pass before merging” and add `validate_modified_declarations` and `validate_schema` as required status checks.
+- In the `Actions` section, in the `General` submenu: select "Allow all actions and reusable workflows".
+
+#### Update readme
+
+- Update the readme file with proper metadata: topic, maintainers, jurisdictions, languages…
+
+#### Update deployment inventory
+
+- Open the `deployment/inventory.yml` and set server information: ip, user, fingerprint…
+
+### Snapshots
+
+Create the snapshot repository by using the [Demo snapshots](https://github.com/OpenTermsArchive/demo-snapshots) as template:
+- Go to the [demo snapshots repository homepage](https://github.com/OpenTermsArchive/demo-snapshots)
+- Click on the "Use this template" dropdown
+- Select "Create a new repository"
+- In the opened page, fill the repository name, it is strongly recommended to use the `<collection_id>-snapshots`.
```

Comment:
It is more than recommended: if the user does not follow this naming pattern, the automated replacements will fail.

```suggestion
- Set the repository name to `<collection_id>-snapshots`.
```